packages/vscode/package.json

```diff
@@ -61,14 +61,30 @@
           "source.js",
           "source.ts",
           "source.js.jsx",
+          "source.jsx",
           "source.tsx",
           "source.vue"
         ],
         "scopeName": "inline.prisma",
-        "path": "./syntaxes/prisma.js.json",
+        "path": "./syntaxes/prisma-inlined.json",
```

Comment:
Good change, that makes it much clearer what this is.

Reply:
Yes, just naming of the file her